intArray.h

```diff
@@ -78,10 +78,4 @@ class IntArray : public Object{
      */
     virtual bool equals(Object *other);
 
-    /*
-    * Checks equality between this array and a given object
-    * @arg other: the other object to check equality to
-    */
-    bool equals(int other);
```

Comment:
Also, there was an extra equals left behind 🙃

Reply:
Thank you for deleting this!